flink-metrics/flink-metrics-core/src/main/java/org/apache/flink/metrics/groups/UnregisteredMetricsGroup.java

```diff
@@ -178,4 +182,35 @@ public T getValue() {
             return null;
         }
     }
+
+    private static class UnregisteredSinkWriterMetricGroup extends UnregisteredMetricsGroup
```

Comment:
Why we need this implementation?

Reply:
Because I didn't find a good way to mock SinkWriterMetricGroup, and I saw there are other UnregisterdXXXMetricGroup() used in Sink